src/accessors/IExecutionResult.ts

```diff
@@ -0,0 +1,11 @@
+import { ILogMessage } from './ILogMessage';
+/**
+ * All executions of pre- and post-exits are enabled to perform the modifications
+ * they aim at themselves. This interface provides a mechanism to propagate information
+ * about whether this execution was actually successful
+ */
+
+export interface IExecutionResult {
+    failed: boolean;
```

Comment:
@graywolf336 `failed` instead of `success` reduces the amount of coded needed if everything goes right - which should be the default ;)